cmake-modules/AzureVcpkg.cmake

```diff
@@ -93,7 +93,7 @@ macro(az_vcpkg_export targetName macroNamePart dllImportExportHeaderPath)
   include(GNUInstallDirs)
 
   # When installing, copy our "inc" directory (headers) to "include" directory at the install location.
-  install(DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/inc/" DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+  install(DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/inc/azure/" DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/azure")
```

Comment:
Will this affect the header include paths of the SDK for customers?

Also, CL entry?

Reply:
The change is invisible to our customers, everything will work exactly as before.